doc/changelog.rst

```diff
@@ -13,6 +13,7 @@ New Features:
 
 - Added the :class:`motor.motor_asyncio.AsyncIOMotorClientEncryption` class,
 with the same interface as the corresponding PyMongo class.
+See :doc:`examples/encryption` for examples.
```

Comment:
Does this render correctly? I think you need to manually add space before continuing the bullet on the next line. See the other multi-line changelog entries for an example. You could also try to build the docs yourself and see if that works.

Reply:
ah you're right, fixed!